### ISSUE TITLE
pac4j-cas: Update or remove dependency on xmlsecurity:xmlsec to remove unnecessary debug output

### ISSUE THREAD

**Issue by millross** [pac4j/pac4j#312]
Version 1.3 of xmlsecurity:xmlsec (on which pac4j-cas has a compile dependency) contains a log4j.properties file which sets the root logger level to debug, leading to lots of debug output in projects which depend on pac4j-cas.

See here https://bz.apache.org/bugzilla/show_bug.cgi?id=45095

PInged Jérôme yesterday and he thinks this dependency is no longer needed. However, if it is still required, there are numerous later versions of xmlsec under the org.apache.santuario groupId (xmlsecurity was dropped quite some time ago, I think), I'm assuming the 1.x API should remain consistent, so we should probably move up to a later version of this dependency if we still do need it.

From Jérôme's email: The truth is that we upgraded to the latest CAS client (v3.4.1) and the XML support is no more based on opensaml (but via the cas-client-support-saml submodule), so I think we can safely remove the xmlsec as well as the commons-codec dependencies.

**Comment by leleuj** [comment]
PR: https://github.com/pac4j/pac4j/pull/313
